bw2io/extractors/simapro_csv.py

```diff
@@ -285,11 +388,83 @@ def get_project_name(cls, data):
 
     @classmethod
     def invalid_uncertainty_data(cls, amount, kind, field1, field2, field3):
+        """
+        Determines if the uncertainty data is invalid.
+
+        Parameters
+        ----------
+        amount : str
+            The amount of uncertainty.
+        kind : str
+            The kind of uncertainty.
+        field1 : str
+            The first field of uncertainty data.
+        field2 : str
+            The second field of uncertainty data.
+        field3 : str
+            The third field of uncertainty data.
+
+        Returns
+        -------
+        bool
+            `True` if the uncertainty data is invalid, `False` otherwise.
+
+        Notes
+        -----
+        This method checks if the given uncertainty data is invalid based on the kind of uncertainty.
+        If the kind is "Lognormal" and `amount` is empty or `field1` is "0" or "1", the uncertainty data is considered invalid.
+        """
         if kind == "Lognormal" and (not amount or field1 == "0" or field1 == "1"):
             return True
 
     @classmethod
     def create_distribution(cls, amount, kind, field1, field2, field3):
+        """
+        Creates a distribution based on the given uncertainty data.
+
+        Parameters
+        ----------
+        amount : str
+            The amount of uncertainty.
+        kind : str
+            The kind of uncertainty.
+        field1 : str
+            The first field of uncertainty data.
+        field2 : str
+            The second field of uncertainty data.
+        field3 : str
+            The third field of uncertainty data.
+
+        Returns
+        -------
+        dict
+            A dictionary representing the distribution.
+
+        Raises
+        ------
+        ValueError
+            If the given uncertainty type is unknown.
+
+        Notes
+        -----
+        This method creates a distribution based on the given uncertainty data.
```

Comment:
How does this render in Sphinx? I encountered some issues with enumerations and lists previously.

Reply:
I adopted your solution and inserted a line between the last sentence and the closing """, hope that will prevent the issue you encountered earlier from appearing.